strings.md

```diff
@@ -1022,6 +1048,12 @@ comma-separated prefix parameters:
 ;; "foo                           "
 ~~~
 
+or to create a string having repititive characters by combining with `~:*`:
+~~~lisp
+(format t "~v@{~A~:*~}" 5 "*")
```

Comment:
so this is like `(make-string 5 :initial-element #\*)`? Then is it worth showing this obscure format directive? (if you feel like yes, then yes)

Reply:
I have deleted it.